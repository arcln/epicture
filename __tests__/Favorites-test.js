import 'react-native';
import React from 'react';
import Favorites from '../screens/FavoritesScreen';
import renderer from 'react-test-renderer';
import NavigationTestUtils from 'react-navigation/NavigationTestUtils';

describe('Favorites snapshot', () => {
  jest.useFakeTimers();
  beforeEach(() => {
    NavigationTestUtils.resetInternalState();
  });

  it('renders the screen', async () => {
    const tree = renderer.create(<Favorites />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
