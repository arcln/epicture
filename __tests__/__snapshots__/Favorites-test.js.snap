// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Favorites snapshot renders the screen 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#ededed",
        "flex": 1,
        "justifyContent": "flex-start",
      }
    }
  >
    <View />
    <View
      style={
        Object {
          "height": 1334,
          "paddingBottom": 50,
        }
      }
    >
      <View
        style={
          Object {
            "flex": 1,
          }
        }
      />
    </View>
  </View>
</View>
`;
